composer.json

```diff
@@ -24,41 +24,12 @@
   ],
   "require": {
     "php": ">=5.3.3",
-    "symfony/symfony": "2.6.*",
-    "doctrine/orm": ">2.4,<2.6",
-    "doctrine/dbal": ">2.4,<2.6",
-    "doctrine/doctrine-bundle": "~1.2",
-    "symfony/assetic-bundle": "~2.3",
-    "symfony/monolog-bundle": "~2.4",
-    "sensio/distribution-bundle": "~3.0,>=3.0.12",
-    "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
-    "incenteev/composer-parameter-handler": "~2.0"
+    "doctrine/dbal": "~2.4"
+  },
+  "suggest": {
+    "doctrine/orm": "To use DQL functions"
```

Comment:
the ORM should be added in the dev requirements though, to be able to run tests for the DQL functions